msu/hooks/skills/skill.nut

```diff
@@ -437,6 +437,17 @@
 
 		return _tooltip;
 	}
+
+// New Events
+	// This event is triggered once for every skill of a player character directly after the onHired event of that player class is triggered
+	q.onHired <- function()
+	{
+	}
+
+	// This event is triggered once for every skill of every other character in the player roster after the onHired event of a player class is triggered
+	q.onOtherPlayerHired <- function( _otherCharacter )
+	{
+	}
```

Comment:
Two points:
- Would it be a better setup is to simply have a single event `onHired( _bro )` or `onAnyBroHired( _bro )`? Then for the case of the bro himself you pass him, and for others you pass the other actor.
- The word `Player` in the function name `onOtherPlayerHired` seems awkward as it has other connotations besides the `player.nut` class. So maybe something like `onOtherBroHired` is better. Other suggestions?

Reply:
I switched to the single event